frappe/public/js/frappe/views/map/map_view.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * frappe.views.MapView
+ */
+frappe.provide("frappe.views");
+
+frappe.views.MapView = class MapView extends frappe.views.ListView {
+	get view_name() {
+		return 'Map';
+	}
+
+	setup_defaults() {
+		super.setup_defaults();
+		this.page_title = __('{0} Map', [this.page_title]);
+	}
+
+	setup_view() {
+	}
+
+	on_filter_change() {
+		this.get_coords();
+	}
+
+	prepare_data(data) {
+		super.prepare_data(data);
+		this.items = this.data.map(d => { return d; });
+	}
+
+	render() {
+		this.get_coords()
+			.then(() => {
+				this.render_map_view();
+			});
+		this.$paging_area.find('.level-left').append('<div></div>');
+	}
+
+	render_map_view() {
+		this.map_id = frappe.dom.get_unique_id();
+
+		this.$result.html(`
+			<div id="` + this.map_id + `" class="map-view-container"  style="width: 100%;height: calc(100vh - 284px); z-index: 0;">
+				
+			</div>
+		`);
+
+
+
+		this.map = L.map(this.map_id).setView([12.3112899, -85.7384542], 8);
```

Comment:
Maybe there is a way to center the map based on points and not fixed coordinates

Reply:
This is a default coordinates of map. It linkes to India

Reply:
@ernestoruiz89 I fixed, can you approve it now?)

Reply:
@AminovE99 Hi, I just a common user, wait Frappe Reviewers